specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,678 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/SessionPool"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Session Pool": {
+            "$ref": "./examples/SessionPools_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ContainerAppsSessionPools"
+        ],
+        "summary": "Create or update a Session Pool.",
+        "description": "Create or update a Session Pool.",
+        "operationId": "ContainerAppsSessionPools_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "sessionPoolName",
+            "in": "path",
+            "description": "Name of the Session Pool.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63
+          },
+          {
+            "name": "sessionPoolEnvelope",
+            "in": "body",
+            "description": "Properties used to create a session pool",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SessionPool"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Create or update succeeded",
+            "schema": {
+              "$ref": "#/definitions/SessionPool"
+            }
+          },
+          "201": {
+            "description": "Session Pool create or update has been started.",
+            "schema": {
+              "$ref": "#/definitions/SessionPool"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Create or Update Session Pool": {
+            "$ref": "./examples/SessionPools_CreateOrUpdate.json"
+          }
+        }
+      },
+      "patch": {
```

Comment:
We don't have this method from CP right? If we don't have plan to support patch at this version from CP, we should remove this, otherwise the SDK would return 404 when call this method.

Reply:
Thanks, I have added the Patch method in that Update SessionPool Apis align with Swagger CP PR (still under review). Since Patch is required for a tracked arm resource https://github.com/Azure/azure-openapi-validator/blob/main/docs/tracked-resource-patch-operation.md

Reply:
+1 Not sure about PATCH operation. Please double confirm this one.

Reply:
You support patch for at least tags right? Since its required.